ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/prior_box.cpp

```diff
@@ -83,6 +83,35 @@ namespace ngraph
                         axes)};
                 }
 
+                OutputVector prior_box_clustered(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    NGRAPH_CHECK(inputs.size() == 2, "Invalid number of inputs");
+
+                    auto output_shape = std::make_shared<default_opset::ShapeOf>(inputs[0]);
+                    auto image_shape = std::make_shared<default_opset::ShapeOf>(inputs[1]);
+                    auto output_shape_slice = detail::make_slice(output_shape, 2, 4);
+                    auto image_shape_slice = detail::make_slice(image_shape, 2, 4);
```

Comment:
What will happen if input will have different rank than 4?

Reply:
@mateusztabaka - what do you think?

Reply:
yes, we can add that check.